### ISSUE TITLE
CLI agrument not working with custom script

### ISSUE THREAD

**Issue by hiepxanh** [ionic-team/ionic-cli#3845]
[Small Feature Request]

**Description:**
Step 1: I create HMR angular project follow this angular cli wiki [angular stories configure hmr](https://github.com/angular/angular-cli/wiki/stories-configure-hmr). If I run `npm run hmr` everything just work fine.

Step 2: I follow this issue [#3819](https://github.com/ionic-team/ionic-cli/issues/3819) to run custom script `ionic serve` matching with `ng serve --configuration hmr`. 

```
  "scripts": {
    "ionic:serve": "ng serve --configuration hmr"
  }
```

I think there is only one small problem is: If I'm passing `--prod` for the command: `ionic cordova build android --prod`. It will not forward to `ionic:build` script. expect that I have to use `ionic cordova build android -- --prod` to make it go production.

**Steps to Reproduce:**

`ionic cordova build android --prod` not run ionic build production.

![image](https://user-images.githubusercontent.com/16970990/52101600-ea12a980-260e-11e9-9e20-e537fb553d9d.png)
![image](https://user-images.githubusercontent.com/16970990/52101606-f139b780-260e-11e9-8b9a-7fef4d92d9ac.png)

`ionic cordova build android -- --prod` is working.

![image](https://user-images.githubusercontent.com/16970990/52101621-00b90080-260f-11e9-8e25-9bca436ec026.png)
![image](https://user-images.githubusercontent.com/16970990/52101630-06aee180-260f-11e9-9a09-7ea81861fede.png)

**Output:**

![image](https://user-images.githubusercontent.com/16970990/52101363-e16da380-260d-11e9-9928-77b592528624.png)

**Conclusion:**
I think it not really important. It just will be more convenient for developer that not have to remember to change anything if they want to use HMR. Thank you so much.

**Comment by dwieeb** [comment]
@hiepxanh This looks like a bug in the Ionic CLI. I just pushed a commit to fix this and it should be in 4.10.2! Thanks!

**Comment by hiepxanh** [comment]
thank you @dwieeb
